doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -0,0 +1,40 @@
+.. _abi_compatibility:
+
+ABI Compatibility
+*****************
+
+.. contents::
+   :local:
+   :depth: 2
+
+Application Binary Interface (ABI) compatibility defines how software components, like libraries and applications, interacts at the machine code level through their low-level binary interface, including:
+
+* Function-calling conventions
+* Data structure layouts in memory
+* Exception handling mechanisms
+* Register usage conventions
+
+When ABI compatibility is maintained, binaries of one component can interface correctly with another without needing recompilation.
+For example, adding a new function to a library is typically an ABI-compatible change, as existing binaries remain functional.
+However, changes that affect data structure layouts, such as altering field order or size, break ABI compatibility as they change the memory layout expected by existing binaries.
+
+ABI Compatibility Matrix for the for the nrf54h20_soc_binary bundle
+===================================================================
+
+The table below illustrates ABI compatibility between different versions of the *nrf54h20_soc_binary* bundle and the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 25
+
+   * - nrf54h20_soc_binary version
+     - Compatible |NCS| versions
+     - Notes
```

Comment:
I think that from an application developer perspective, this table should be structured the other way around:

| NCS version | nrf54h20_soc_binary compatible with NCS version |

The entry point for an application developer is the NCS version they select for their application. The required nrf54h20_soc_binary is a result of their NCS version choice.

Reply:
Should we also mention the compatible nRF4H20 DK version?

Reply:
Hopefully not. When FP1.1 gets released all DKs should be equally compatible with the soc_bundle.

But someone from Aurora and Tarpand should confirm my hopes :)

Reply:
@hakonfam FYI